### PR TITLE
Private endpoints how-to

### PR DESCRIPTION
**Pull request by matkuliak**
## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by matkuliak** [comment]
Requires https://github.com/crate/cloud-howtos/pull/71 to be merged first

File: docs/private-endpoints.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+If you're interested in this feature, `contact us`_ and we'll gladly get you
+started. The workflow will look something like this:
+
+1. Contact us and ask about configuring a private endpoint. We'll ask for your
+   AWS Account ID or Azure Subscription ID.
+2. We will create the endpoint service and provide you with the Alias/ID.
+3. You create the private endpoint request.
+4. We confirm it.
+
+That's it. After this, you will have a secure, private way of connecting to
```
Comment (SStorm) [inline docs/private-endpoints.rst:34]: Do we want to mention that this service is free?

**Review by SStorm — APPROVED** [review]

File: docs/private-endpoints.rst (hunk 1 above)
Comment (matkuliak) [reply docs/private-endpoints.rst:34]: Do you mean from our side, or from AWS/Azure ?

Comment (SStorm) [reply docs/private-endpoints.rst:34]: Our side. This is just a thought, feel free to merge it as is.

Comment (matkuliak) [reply docs/private-endpoints.rst:34]: I see. Yeah, I think I'll keep it like this for now. To me, it reads like it is a standard feature that comes with the product, no premium. I could be biased of course. If you ever strongly feel like it should be mentioned, let me know, thanks!

**Pull request merged**